core/src/main/java/org/folio/tlib/util/TenantUtil.java

```diff
@@ -8,7 +8,7 @@
   // PostgreSQL names are case insensitive and must not start with a digit.
   // The maximum length is 63 characters, schema = tenant + '_' + moduleName
   // where tenant and moduleName length have a maximum of 31 each.
-  private static final String TENANT_PATTERN_STRING = "^[a-z][a-z0-9]{0,30}$";
+  private static final String TENANT_PATTERN_STRING = "^[a-z][a-z0-9]*(_[0-9]+)*$";
```

Comment:
It should accept every tenant that Okapi and PostgreSQL currently accept:
"^[_a-z][_a-z0-9]*$"

More restriction can be imposed at a later time.

Reply:
ok